components/drops/useDrops.ts

```diff
@@ -83,7 +78,11 @@ export function useDrops(query?: GetDropsQuery) {
 
 export const getFormattedDropItem = async (collection, drop: DropItem) => {
   const chainMax = collection?.max ?? FALLBACK_DROP_COLLECTION_MAX
-  const { count } = await getDropStatus(drop.alias)
+
+  let count = drop.minted
+  if (!count) {
+    count = await fetchDropMintedCount(drop)
+  }
```

Comment:
if `drop.minted` is 0 which would be valid , wouldn't this also call `fetchDropMintedCount` unnecessarily?, 

or is this by design.

Reply:
The `drop.minted` would always be `0` in the new mint flow, while some old drops would return valid `drop.minted` which is greater than 0

ref: https://github.com/kodadot/private-workers/issues/99